chainer/dataset/tabular/_concat.py

```diff
@@ -108,3 +108,6 @@ def get_examples(self, indices, key_indices):
                     [examples[dataset_index][col_index][p]
                      for dataset_index, p in example_indices]
                     for col_index in six.moves.range(n_cols))
+
+    def convert(self, data):
+        return self._datasets[0].convert(data)
```

Comment:
It should be documented that the converter of the first dataset is used in `concat` and `join`. The docstring of `TabularDataset.convert` may be a good place to add a note.